ios/src/TGMapViewController.h

```diff
@@ -12,6 +12,26 @@
 #import <UIKit/UIKit.h>
```

Comment:
From my understanding, that's because I did a rebase from tangrams/master to hallahan/minZoom. Git gets kinda weird when dealing with another origin sometimes. It's because I did commits at a point before this iOS work and then rebased yesterday. The nature of rebase is that history gets rewritten, so that commit, as you can see, has a different SHA1. The older problem where we saw old commits that have already been merged is because I branched off of hallahan/master instead of tangrams-es/master. It was up to date, but the fact that one origin merged into another created that effect.

The only way I could resolve this is to make a new branch and cherry pick my commits into a new one and open a new PR. However, I wonder, would it take me out of the commit message once you merge the PR? Since, all of my commits should say it was hallahan with tallytalwar? I can do a new PR if needed. Sorry this is such a headache!

http://stackoverflow.com/questions/32826197/interactive-rebase-after-merging-other-commits-interleaving-mine